src/components/TextField/TextField.tsx

```diff
@@ -3,11 +3,17 @@ import React, { InputHTMLAttributes } from 'react';
 import style from './TextField.module.scss';
 
 type TextFieldProps = {
+  label: string;
   onChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
 } & InputHTMLAttributes<HTMLInputElement>;
 
-export const TextField: React.FC<TextFieldProps> = ({ onChange, ...props }) => (
+export const TextField: React.FC<TextFieldProps> = ({
+  onChange,
+  label,
+  ...props
+}) => (
   <label>
+    <span className={style.label}> {label} </span>
```

Comment:
Are the spaces intentional?

```suggestion
    <span className={style.label}>{label}</span>
```

Reply:
no